pkg/cmd/dev/testlogic.go

```diff
@@ -120,13 +118,6 @@ func (d *dev) testlogic(cmd *cobra.Command, commandLine []string) error {
 		return err
 	}
 
-	if !noGen {
```

Comment:
Not 100% sure about deleting this flag. I think this functionality is still useful.

One option if you find the `--no-gen` thing annoying is to add support for an environment variable (like `DEV_NO_GENERATE_LOGICTEST`) so you can set it in your profile. We already support environment variables like `DEV_NO_REMOTE_CACHE` for turning off functionality globally.

Reply:
Do you think it's still useful for `dev test` too, or just `dev logictest`?

Reply:
I can go either way on `dev test`, definitely I think keep it for `testlogic` though.

Reply:
SGTM. I added it back to just `testlogic` for now.